homeassistant/components/velux/cover.py

```diff
@@ -97,12 +99,13 @@ async def async_open_cover(self, **kwargs):
 
     async def async_set_cover_position(self, **kwargs):
         """Move the cover to a specific position."""
-        if ATTR_POSITION in kwargs:
-            position_percent = 100 - kwargs[ATTR_POSITION]
+        if ATTR_POSITION not in kwargs:
+            return
```

Comment:
Is this guard needed ? It looks like the schema makes this unreachable unless there is something else calling it?

https://github.com/home-assistant/core/blob/7f0091280fe6c8b116bc7abae6cd5cca30f2b28d/homeassistant/components/cover/__init__.py#L139

Reply:
No idea, it was already there so I'm scared to delete it without knowing further details

Reply:
Removed the redundant guard. Nothing else was calling that method